src/frontend/efiling-demo/src/__snapshots__/storyshots.test.js.snap

```diff
@@ -306,6 +306,324 @@ exports[`Storyshots Cancel Mobile 1`] = `
+          >
+            <a
+              className="nav-link"
+              href="https://www2.gov.bc.ca"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              BC Government
+            </a>
+          </li>
+          <li
+            className="nav-item m-1"
+          >
+            <a
+              className="nav-link"
+              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Disclaimer
+            </a>
+          </li>
+          <li
+            className="nav-item m-1"
+          >
+            <a
+              className="nav-link"
+              href="https://www2.gov.bc.ca/gov/content/home/privacy"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Privacy
+            </a>
+          </li>
+          <li
+            className="nav-item m-1"
+          >
+            <a
+              className="nav-link"
+              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Accessibility
+            </a>
+          </li>
+          <li
+            className="nav-item m-1"
+          >
+            <a
+              className="nav-link"
+              href="https://www2.gov.bc.ca/gov/content/home/copyright"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Copyright
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </footer>
+</main>
+`;
+
+exports[`Storyshots Error Mobile 1`] = `
+<main>
+  <header>
+    <nav
+      className="container-fluid navbar navbar-expand-lg navbar-dark"
+    >
+      <div
+        aria-labelledby="title"
+        className="navbar-brand pointer"
+      />
+      <img
+        alt="B.C. Government Logo"
+        className="img-fluid d-none d-md-block pointer"
+        height="44"
+        onClick={[Function]}
+        onKeyDown={[Function]}
+        role="presentation"
+        src="undefined/images/bcid-logo-rev-en.svg"
+        width={181}
+      />
+      <img
+        alt="B.C. Government Logo"
+        className="img-fluid d-md-none pointer"
+        height="44"
+        onClick={[Function]}
+        onKeyDown={[Function]}
+        role="presentation"
+        src="undefined/images/bcid-symbol-rev.svg"
+        width={64}
+      />
+      <div
+        aria-labelledby="title"
+        className="pointer navbar-brand nav-item nav-link"
+      />
+      <div
+        className="navbar-brand"
+        id="title"
+      >
+        eFiling Demo Client
+      </div>
+    </nav>
+  </header>
+  <div
+    className="page"
+  >
+    <div
+      className="content col-md-10"
+    >
+      <p>
+        The following error occurred with your eFiling application. Please click the button below to go back home.
```

Comment:
with your submission? I'm not sure but eFiling application sounds a little weird here. I could be wrong. Non-blocking.